components/brave_adaptive_captcha/server_util.cc

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_adaptive_captcha/server_util.h"
+
+#include <utility>
+
+#include "base/check.h"
+#include "base/command_line.h"
+#include "base/notreached.h"
+#include "brave/components/brave_ads/common/switches.h"
+
+namespace {
+
+const char kDevelopment[] = "https://grant.rewards.brave.software";
```

Comment:
I believe the current policy is not to have default urls hardcoded cc @jumde

Reply:
Updated to retrieve URLs from npm config settings and created https://github.com/brave/devops/issues/5767 to update Jenkins with those settings.